BenadrylAutomation/src/main/java/com/jnj/actions/Action.java

```diff
@@ -1004,30 +1004,16 @@ public static void navigateBack() {
 	public static void navigateForward() {
 		driver.navigate().forward();
 	}
-
+	
 	/**
 	 * Function to verify page url
```

Comment:
@zahinSJI there should be empty line after this. i can see you have removed that, pls add again

Reply:
There is an empty line after every new function
![image](https://github.com/sji-autobots/BenadrylAutomation/assets/107922907/6068362a-58a6-4cd1-8aa2-3bda0c9cfd24)